stdlib/3/multiprocessing/spawn.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, List, Optional
+from types import ModuleType
+
+WINEXE: bool
+WINSERVICE: bool
+# undocumented
+_python_exe: str
+
+def set_executable(executable: str) -> None: ...
+def get_executable() -> str: ...
+def is_forking(argv: List[str]) -> bool: ...
+def freeze_support() -> None: ...
+def get_command_line(**kwds: Any) -> List[str]: ...
+def spawn_main(pipe_handle: int, parent_pid: Optional[int] = ..., tracker_fd: Optional[float] = ...) -> None: ...
```

Comment:
Is `tracker_fd` really a `float`? Just going by the argument name, I'd expect an `int`.

Reply:
Yep this is obviously a file descriptor, no idea what happened here